src/index.html

```diff
@@ -52,9 +56,37 @@ <h2><span class="Datateknik">Datateknik</span> && <span class="InfoCom">Informat
       </p>
     </header>
     <div class="main-content"></div>
-    <div class="footer">
-      <button id="feedback-form">Rapportera problem med sidan</button>
-    </div>
+    <footer>
+      <div class="content">
+        <p>D-sektionen inom TLTH</p>
+        <p><a href="#">Kontakta oss</a></p>
+        <div class="address">
+          Besöksadress:<br>
+          E-huset<br>
+          Ole Römers väg 3<br>
+          22363 LUND
+        </div>
+        <div class="address">
+          Postadress:<br>
+          D-sektionen inom TLTH <br>
+          Box 118 <br>
+          22100 LUND
+        </div>
+        <p>Organisationsnummer: 845003-2878</p>
+        <button id="feedback-form">Rapportera problem med sidan</button>
```

Comment:
If this is in swedish maybe also the form-text should be?
Also, do we want the wording to be for problems of for feedback, i think there is a difference

Reply:
I agree, I will change the form to Swedish as well as switching it from problems to feedback.

Reply:
It doesn't seem as though it is possible to change the field text to swedish

Reply:
I looked at zammad, there doesn't seem to be a way to change it there either.
If we really care i think we might be able to change the source code, but that might be overkill 🤷‍♂